File: Eticaret2Csharp/Siparis.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Eticaret2_
{
    public class Siparis
    {
        public static void OdemeYap(Kullanici kullanici)
        {
            double toplamTutar = 0;

            foreach (Urun urun in kullanici.GetUrunler())
            {
                toplamTutar += urun.GetFiyat();
            }

            toplamTutar = kullanici.GetToplamTutar(toplamTutar);

            System.Console.WriteLine(kullanici.GetType().Name + " Toplam Tutar = " + toplamTutar);
        }
    }

}

File: E-Ticaret/E-ticaret2py/Siparis.py
from Urun import Urun
from Kullanici import Kullanici

class Siparis:
    @staticmethod
    def odeme_yap( kullanici):
        toplam_tutar=0
        for urun in kullanici.get_urunler():
            toplam_tutar += urun.get_fiyat()
        toplam_tutar = kullanici.get_toplam_tutar(toplam_tutar)
        print(f"{kullanici.__class__.__name__} Toplam Tutar = {toplam_tutar}")
     #print(f"{kullanici.__class__.__name__} Toplam Tutar = {toplam_tutar}")
